platform/vs/docker-physyncd-vs/supervisord.conf

```diff
@@ -0,0 +1,28 @@
+[supervisord]
+logfile_maxbytes=1MB
+logfile_backups=2
+nodaemon=true
+
+[program:start.sh]
+command=/usr/bin/start.sh
+priority=1
+autostart=true
+autorestart=false
+stdout_logfile=syslog
+stderr_logfile=syslog
+
+[program:rsyslogd]
+command=/usr/sbin/rsyslogd -n
```

Comment:
This now needs to change to `command=/usr/sbin/rsyslogd -n -iNONE`, and you should also add lines like:

```
dependent_startup=true
dependent_startup_wait_for=...
```

Please refer to the latest https://github.com/Azure/sonic-buildimage/blob/master/platform/vs/docker-syncd-vs/supervisord.conf as an example.